doc/sphinx-guides/source/developers/dependencies.rst

```diff
@@ -240,13 +244,13 @@ Maven can manage versions of transitive dependencies in four ways:
     * - Safe Good Practice
       - (1) Explicitly declare the transitive dependency in ``<dependencyManagement>`` with a ``<version>`` tag.
       - (2) For more complex transitive dependencies, reuse a "Bill of Materials" (BOM) within ``<dependencyManagement>``.
-            Many bigger and standard use projects provide those, making the POM much less bloated compared to adding every bit yourself.
+            Many bigger and standard use projects provide them, making the POM much less bloated compared to adding every bit yourself.
```

Comment:
What are "standard use projects"? Maybe we could simplify this and just say "bigger projects".

Reply:
```suggestion
            Many bigger projects provide them, making the POM much less bloated compared to adding every bit yourself.
```